.github/workflows/dev.yaml

```diff
@@ -54,10 +54,12 @@ jobs:
         run: cat run_result 2>/dev/null || echo 'not run'
```

Comment:
These changes are incorrect but they aren't used since the dev.yaml uses the `.github/workflows/dev.yaml` from the branch we are merging into.